backend/typescript/services/implementations/EntityServiceMg.ts

```diff
@@ -100,6 +148,7 @@ class EntityService implements IEntityService {
       if (!deletedEntity) {
         throw new Error(`Entity id ${id} not found`);
       }
+      this.storageService.deleteFile(deletedEntity.fileName);
```

Comment:
This throws an error if `deletedEntity` doesn't have an associated file to begin with right? Don't think it should be an error condition, let it silently fail

Reply:
Do you mean I should update the FileStorageService or add another try catch in this function for that case?

Reply:
I think either another try-catch, or wrapping it in `if (deletedEntity.fileName) { ... }` are fine

Reply:
Actually let's just do the if statement. If the `filename === ""`, that means there isn't an associated file and we should not even attempt to delete and log an error. If there is a `filename` but it doesn't exist in storage for some reason, that's an error case (this case is already accounted for here).